freeCodeCamp/fcctesting.js

```diff
@@ -38,6 +38,7 @@ const fs = require("fs");
 
 const allowedOrigins = [
   /^https?:\/\/([\w-]+\.)*freecodecamp.org/,
+  /^https:\/\/([\w-]+\.)*gitpod.io/,
```

Comment:
What is the purpose of adding this line?  I may have missed something in the discussion of why it is being added.

Reply:
In the case someone wants to do this project in Gitpod - me while testing. Gitpod does not expose `localhost` - it uses https subdomains.